cpp/src/command_classes/MultiInstance.cpp

```diff
@@ -147,6 +148,22 @@ void MultiInstance::ReadXML
 	{
 		m_uniqueendpoints = !strcmp( str, "true");
 	}
+	str = _ccElement->Attribute("version");
```

Comment:
I think this should be done in the CommandClass class then it can be used for any other CC's that might have this issue. Can you move this?